experiments/opencog/pattern_matcher_vqa/interface.py

```diff
@@ -1,18 +1,23 @@
 from abc import ABC
 
+
 class NeuralNetworkRunner(ABC):
     
     def runNeuralNetwork(self, features, word):
         pass
 
+
 class AnswerHandler(ABC):
     
     def onNewQuestion(self, record):
         pass
     
     def onAnswer(self, record, answer):
         pass
-    
+
+    def get_unanswered(self):
```

Comment:
I would keep code style consistent in the project modules to improve code readability.